.circleci/config.yml

```diff
@@ -18,6 +18,7 @@ jobs:
                   name: yarn
                   command: yarn --frozen-lockfile install || yarn --frozen-lockfile install
             - run: yarn build:ci:no_website
+            - run: yarn build:ts
```

Comment:
Will that build twice on CI and increase the running time of all tests?

Reply:
Yes, but only by about 1-2 seconds. The beauty of project references is that `build:ts` is pretty much instant if there are no changes to any TypeScript code 😄

Reply:
This is from the latest CI run:

<img width="1498" alt="screen shot 2018-12-11 at 4 46 12 pm" src="https://user-images.githubusercontent.com/800857/49839538-520de780-fd64-11e8-8a33-4e4e8cea8dc4.png">